content/administration/maintain/azure_oauth.rst

```diff
@@ -0,0 +1,170 @@
+   :alt: Link to the created application and to add users/groups.
+
+In the :guilabel:`Left Menu` select :guilabel:`Users and Groups`. Next select the :guilabel:`(+)
+Add User/Group`. Depending on the account, either a :guilabel:`Group` and :guilabel:`User` can be
+added, or :guilabel:`Users` alone. Personal accounts will only allow for users to be added.
+
+Add the users or group of users that will be sending from the :guilabel:`Microsoft account` in
+:guilabel:`Odoo` by clicking on :guilabel:`None Selected` under users or groups. :guilabel:`Add`
+the users/groups, click :guilabel:`Select` and then :guilabel:`Assign` them to the application.
+
+This completes the setup on the :guilabel:`Microsoft Azure Portal` side.
+
+Setup in Odoo
+=============
+
+Enter Microsoft Outlook Credentials
+-----------------------------------
+
+First, open :guilabel:`Odoo` and navigate to the :guilabel:`Apps module`. Then, remove the Apps
+filter from the :guilabel:`search bar` and type in `Outlook`. Install the module called
+:guilabel:`Microsoft Outlook`.
+
+Next, navigate to :guilabel:`Settings ‣ General Settings`, and under the :guilabel:`Discuss`
+section, ensure that the checkbox for :guilabel:`Custom Email Servers` is checked. This populates a
+new option for :guilabel:`Outlook Credentials`.
+
+:guilabel:`Save` the progress.
+
+Then, copy and paste the :guilabel:`Client ID (Application ID)` and :guilabel:`Client Secret
+(Client Secret Value)` into the respective fields and :guilabel:`Save` the settings.
+
+.. image:: azure_oauth/outlookcreds.png
+   :align: center
+   :alt: Outlook Credentials in Odoo General Settings.
+
+Configure outgoing email server
+-------------------------------
+
+After that, on the :guilabel:`General Settings` page, under :guilabel:`Custom Email Servers`, click
+:guilabel:`Outgoing Email Servers` to configure the external :guilabel:`Microsoft account`.
+
+Then, create a new email server and select the option for :guilabel:`Outlook`. Next, fill in the
+:guilabel:`Name` (it can be anything) and the Microsoft Outlook email :guilabel:`Username`. If
+empty, enter the :guilabel:`From Filter`.
+
+Click on :guilabel:`Connect your Outlook account`.
+
+A new window from Microsoft opens to complete the :guilabel:`authorization process`.
+:guilabel:`Select` the appropriate email address that is being configured in Odoo.
+
+.. image:: azure_oauth/verify-outlook.png
+   :align: center
+   :alt: Permission page to grant access between newly created app and Odoo.
+
+Then, allow Odoo to access the :guilabel:`Microsoft account` by clicking on :guilabel:`Yes`. After
+this, the page will navigate back to the newly configured :guilabel:`outgoing email server` in
+Odoo. The configuration automatically loads the :guilabel:`token` in Odoo, and a tag stating
+:guilabel:`Outlook Token Valid` appears in green.
+
+.. image:: azure_oauth/outlook-token.png
+   :align: center
+   :alt: Permission page to grant access between newly created app and Odoo.
```

Comment:
```suggestion
```

Nitpick: this PR is targeting the `14.0` branch, and in Runbot 14.0, the "Name" field is called "Description." I would make note of this so you can open a new PR to correct it for 15.0 user docs, after this PR is merged and forward-ported.